stable/elasticsearch-exporter/values.yaml

```diff
@@ -159,3 +159,15 @@ prometheusRule:
     #   annotations:
     #     description: The heap usage is over 90% for 15m
     #     summary: ElasticSearch node {{$labels.node}} heap usage is high
+
+# Create a service account by default
```

Comment:
i think this comment is a bit misleading. Maybe we can drop "by default"?

Reply:
I've removed "by default" as suggested.